.github/workflows/msvc-full-features.yml

```
@@ -111,7 +109,7 @@ jobs:
         # double-dash after compiler is not a typo, it is to disambiguate between g++-<date> and g++-11-<date> for restore key prefix matching
         key: ccache-${{ github.ref_name }}-${{ runner.os }}-msvc--${{ steps.get-vars.outputs.datetime }}
         restore-keys: |
-          ccache-master-${{ runner.os }}-msvc--
+          ccache-0.H-build-${{ runner.os }}-msvc--
```

```suggestion
          ccache-0.H-${{ runner.os }}-msvc--
```
The dashes are handled in a weird way so let's not add another dash